Build/setup_glass.py

```diff
@@ -118,15 +118,18 @@ def get_glass():
     # Get the latest drop that matches the glass prefix
     get_drop("Temp/DevDiv/Concord", glass_dir, lambda d: "GlassStandalone" in d)
 
+    
+def copy_glass():
     # Copy the Glass.TestAdapter.dll into the extensions directory so vstest.console.exe can find it
     glass_test_adapter = os.path.join(glass_dir, "Glass.TestAdapter.dll")
     if not os.path.exists(glass_test_adapter):
         print(f"Error: Glass.TestAdapter.dll not found at {glass_test_adapter}")
     glass_extensions_dir = os.path.join(glass_dir, "extensions")
-    os.makedirs(glass_extensions_dir, exist_ok=True)
+    if os.path.exists(glass_extensions_dir):
+        shutil.rmtree(glass_extensions_dir)
+    os.makedirs(name=glass_extensions_dir, exist_ok=True)
```

Comment:
we are now caching glass_dir.. so delete sub dir 'glass_dir/extensions' before creating one and moving stuff